src/main/java/core/basesyntax/service/impl/ReportServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static core.basesyntax.db.Storage.storage;
+
+import core.basesyntax.service.ReportService;
+import java.util.Map;
+
+public class ReportServiceImpl implements ReportService {
+    private static final String TITLE = "fruit,quantity";
+    private static final String INDENTATION = "    ";
```

Comment:
Not the best way to use many whitespaces, you can use tabulation or something else